lib/BlocksRegion.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+    import { clean } from './shared';
+
+    export let name;
+    export let blocks;
+</script>
+
+{#if blocks.length}
+    <div class={name}>
+        {#each blocks as block}
+            <div class="block block-{block.id}" class:export-text={block.exportText}>
```

Comment:
I noticed that @elanals changed the block class a little in [her PR](https://github.com/datawrapper/chart-core/pull/20/files#diff-03215a87471532d903240fec4c4cca73R318) 

```diff
- block-{block.id}
+ {block.id}-block
```

Just wanted to make everyone aware that the change doesn't get lost in a future rebase or merge.

Reply:
good point. I have a semantic preference or the `block-{block.id}` version but it's probably causing too much refactoring hazzle for now.